### DISCUSSION TITLE
[Enhancement] Extra Transparency Fix options

### DISCUSSION THREAD

**Discussion by yamanekoayf** [Phobos-developers/Phobos#1063] [Ideas and feedback]
### Description

While current `FixTransparencyBlitters=no` allows disabelling the transparency fix as a whole, there are cases where only a selected few units/structures needed the fix to be lifted for say better visibility in game. 

It would be handy if there is a tag allowing the transparency fix to be lifted in per unit/anim/particle basis

### INI code

```ini
[SOMEANIM]
Anim.UseTransparencyFix=no

[SOMETECHNO]
Unit.UseTransparencyFix=no

[SOMEPARTICLESYS]
Particle.UseTransparencyFix=no
```

### Checklist

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this topic.
- [X] I used the GitHub search and read the topic list to find a similar topic and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

**Comment by Otamaa** [comment]
I think untill someone can write all those code with proper `c++` there is no way to achieve this , and i remember secsome already attempt that but no one cares .

**Reply by yamanekoayf** [reply]
Thanks for explaining the difficulties! Gotta cross my finger for that then!

**Reply by Metadorius** [reply]
What specifically are you referring to and what do you mean by "no one cares"?

**Reply by Otamaa** [reply]
> What specifically are you referring to and what do you mean by "no one cares"?

i mean , he already wrote one on `c++` idk if it is correct one , because i dont understand 
but , no one seems have interest on it , so i say `no one cares` 
https://discord.com/channels/338055131666972683/773636942775582720/1043176739095986207

**Reply by Metadorius** [reply]
Ah this one. No, this has little relation to the suggestion IMO. Instead should be able to just specify another translucency level individually, thus swapping the blitter with the more opaque one.

**Reply by secsome** [reply]
What transparency fix? IIRC all blitter functions had been updated already and Apollo's previous just hooked some of the trans blitters in fact

**Reply by Metadorius** [reply]
What do you mean by "had been updated"?

I think what Otamaa suggests is either fully replacing the blitter function or even replacing the usage of the blitter with the fixed clean implementation.
